tests/models/test_poromechanics.py

```diff
@@ -523,7 +521,7 @@ class PoromechanicsWell(
     well_models.OneVerticalWell,
     porepy.applications.md_grids.model_geometries.OrthogonalFractures3d,
     well_models.BoundaryConditionsWellSetup,
-    pp.poromechanics.Poromechanics,
+    models.Poromechanics,
```

Comment:
Shouldn't this be pp.Poromechanics, since it used to be pp.poromechanics.Poromechanics?

Reply:
Fixed, thanks